src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs

```diff
@@ -0,0 +1,485 @@
+
+        private List<BitChoiceGroupOption> Example2Options;
+
+        private List<BitChoiceGroupOption> Example3Options;
+
+        private List<BitChoiceGroupOption> Example4Options;
+
+        private List<BitChoiceGroupOption> Example7Options;
+
+        /// <summary>
+        /// BitChoiceGroupDemo class constructure
+        /// </summary>
+        public BitChoiceGroupDemo()
+        {
+            Example_1_5_6_Options = new()
+            {
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option A",
+                   Value = "A"
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Option B",
+                    Value = "B"
+                },
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option C",
+                   Value = "C"
+                },
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option D",
+                   Value = "D"
+                }
+            };
+
+            Example2Options = new()
+            {
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option A",
+                   Value = "A"
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Option B",
+                    Value = "B"
+                },
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option C",
+                   Value = "C",
+                   IsEnabled = false
+                },
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option D",
+                   Value = "D"
+                }
+            };
+
+            Example3Options = new()
+            {
+                new BitChoiceGroupOption()
+                {
+                   Text = "Bar",
+                   Value = "Bar",
+                   ImageSrc= "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-bar-unselected.png",
+                   SelectedImageSrc = "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-bar-selected.png",
+                   ImageAlt = "alt for Bar image",
+                   ImageSize = new System.Drawing.Size(32, 32)
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Pie",
+                    Value = "Pie",
+                    ImageSrc= "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-pie-unselected.png",
+                    SelectedImageSrc = "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-pie-selected.png",
+                    ImageAlt = "alt for Pie image",
+                    ImageSize = new System.Drawing.Size(32, 32)
```

Comment:
Plz, using `System.Drawing` in this file and remove in this line

Reply:
what?